app/ante/eth.go

```diff
@@ -191,7 +191,8 @@ func (avd AccountVerificationDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, s
 
 	acc := avd.ak.GetAccount(ctx, address)
 	if acc == nil {
-		return ctx, fmt.Errorf("account %s (%s) is nil", common.BytesToAddress(address.Bytes()), address)
+		acc = avd.ak.NewAccountWithAddress(ctx, address)
```

Comment:
The reason why the account is set to store instead of creating an error should be documented on the comments for future reference 👍